pkg/rm/rm_remoting.go

```diff
@@ -123,6 +130,23 @@ func isReportSuccess(response interface{}) message.ResultCode {
 	return message.ResultCodeFailed
 }
 
+// branchReportResultDecode analyse response result
+func branchReportResultDecode(response interface{}, err error) error {
```

Comment:
The meaning of the method name is decoding, but the method logic is to judge whether the result meets the expectation!

 suggested to put the `err != nil` judgment outside, only the ResultCode is judged here